diff -urN libspf2-1.2.5.org/configure.ac libspf2-1.2.5/configure.ac
--- libspf2-1.2.5.org/configure.ac	2005-02-24 03:09:34.000000000 +0100
+++ libspf2-1.2.5/configure.ac	2005-07-01 20:22:58.647887912 +0200
@@ -126,9 +126,9 @@
 dnl [AC_CHECK_LIB(resolv, res_query)])
 
 # Checks for libraries.
-AC_CHECK_LIB(nsl, inet_pton)
-AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(intl, gettext)
+AC_SEARCH_LIBS(nsl, inet_pton)
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(intl, gettext)
 AC_CHECK_LIB(pthread, pthread_create)
 
 
